python/ray/experimental/serve/tests/test_persistence.py

```diff
@@ -0,0 +1,37 @@
+import os
+import subprocess
+import tempfile
+
+import ray
+from ray.experimental import serve
+
+
+def test_new_driver(serve_instance):
+    script = """
+import ray
+ray.init(address="auto")
```

Comment:
`auto` for now. If travis shows there are existing ray cluster running, I will use `ray.worker._global_node.address_info` to retrieve current gcs location.